### PR TITLE
Show error message for internal errors

### PR DESCRIPTION
**Pull request by kevinxucs**
Show somewhat "meaningful" error message for internal errors, instead of "Unknown or undocumented error"

### CODE REVIEW

File: Goldleaf/Program/ExeFs/main/Include/gleaf/err/Error.hpp [hunk 1]
```diff
@@ -64,6 +64,13 @@ namespace gleaf::err
             "Standard C error (from errno value)",
             "Not enough size available",
 
+            // Goldleaf errors
+            "Not enough space (make sure the device has free space two times the item size)",
+            "Meta not found",
+            "CNM not found",
```
Comment (switch-drama) [inline Goldleaf/Program/ExeFs/main/Include/gleaf/err/Error.hpp]: Should be "CNMT"

Comment (kevinxucs) [reply Goldleaf/Program/ExeFs/main/Include/gleaf/err/Error.hpp]: updated

**Comment by XorTroll** [comment]
I was waiting until everything else was finished to start adding all the missing error descriptions and their translations.

**Comment by XorTroll** [comment]
Added on latest commits, so closing this.

**Pull request closed without merging**